I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 486, in script
    "Cypher: [postacie[Cypher]], Kałach: [postacie[Kalach]], Gun: [postacie[Gun]], Hartmann: [postacie[Hatrmann]], Łaskawca: [postacie[Laskawca]]"
KeyError: 'Hatrmann'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 486, in script
    "Cypher: [postacie[Cypher]], Kałach: [postacie[Kalach]], Gun: [postacie[Gun]], Hartmann: [postacie[Hatrmann]], Łaskawca: [postacie[Laskawca]]"
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\exports.py", line 1474, in say
    who(what, *args, **kwargs)
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\character.py", line 1337, in __call__
    what = self.prefix_suffix("what", self.what_prefix, what, self.what_suffix)
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\character.py", line 1251, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\character.py", line 1231, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\substitutions.py", line 278, in substitute
    s = formatter.vformat(s, (), kwargs) # type: ignore
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/string.py", line 165, in vformat
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/string.py", line 205, in _vformat
  File "C:\Users\kamil\Desktop\renpy-8.1.1-sdk\renpy\substitutions.py", line 168, in get_field
    obj, arg_used = super(Formatter, self).get_field(field_name, args, kwargs)
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/string.py", line 278, in get_field
KeyError: 'Hatrmann'

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
CyberDzban 0.0000000002
Sun Nov 19 13:01:45 2023
